tests/integration/acp/p2p/create_test.go

```diff
@@ -130,3 +130,147 @@ func TestACP_P2PCreatePrivateDocumentsOnDifferentNodes_SourceHubACP(t *testing.T
 
 	testUtils.ExecuteTestCase(t, test)
 }
+
+func TestACP_P2PCreatePrivateDocumentsOnDifferentNodes_SourceHubACP1(t *testing.T) {
+	expectedPolicyID := "fc56b7509c20ac8ce682b3b9b4fdaad868a9c70dda6ec16720298be64f16e9a4"
+
+	test := testUtils.TestCase{
+		Description: "Test acp, p2p create private documents on different nodes, with source-hub",
+		SupportedACPTypes: immutable.Some(
+			[]testUtils.ACPType{
+				testUtils.SourceHubACPType,
+			},
+		),
+		Actions: []any{
```

Comment:
question: so the only thing that makes this test different from the KMS ones is that you don't activate KMS here, right?

Reply:
That seems to be the most important difference yes.